Feature/basket redesign

YelyzavetaDemchenko opened the pull request:
This summarizes work done so far adding redesign for desktop version of Basket.

Ura427 committed:
Made header design for mobile layout

Ura427 committed:
KAN-35 remove comments, fix image, fix layout

Ura427 committed:
KAN-33 remove comments, add responsivenes to small screens via media queries

Ura427 committed:
remove unused import

Yura-Kost committed:
Merge branch 'team_lambda' into refactor/KAN-31/redesign

YelyzavetaDemchenko committed:
refaktor/KAN-32-31/added redux-toolkit

Added work done so far implementing redux-toolkit to this branch

YelyzavetaDemchenko committed:
feature-refaktor/KAN-32-31

Adds header styles for mobile, integrates redux code from previous branch

YelyzavetaDemchenko committed:
redesign/KAN-32/header

Made header sticky.

YelyzavetaDemchenko committed:
feature/KAN-41/header/hero/productList/productCard

Added mobile versions of hero, productList and productCard.

YelyzavetaDemchenko committed:
KAN-41-32/adaptivity and redux basket logic

Adds redux logic for discounts, vouchers, home page for mobile version.

YelyzavetaDemchenko committed:
refaktor/KAN-41

Added responsive design for product list grid.

YelyzavetaDemchenko committed:
feature/basketRedesign

Replaced Drawer component with Modal in order to set offset.

YelyzavetaDemchenko committed:
feature/basketRedesign

Adds redesign to product item in the basket.

YelyzavetaDemchenko committed:
feature/basketRedesign

Adds some basket discount-counter logic.

YelyzavetaDemchenko committed:
feature/basketRedesign

Adds infoToPay section redesign.

YelyzavetaDemchenko committed:
feature/basketRedesign

Adds OrderInfo section redesign.

YelyzavetaDemchenko committed:
feature/basketRedesign

Adds basket redesign for desktop version of application.

VolodymyrKholiavko commented on src/App/AppContainer.tsx:
```diff
@@ -2,7 +2,7 @@ import AnimatedAppearance from '../components/AnimatedAppearance'
 import AppHeader from './AppHeader'
 import Routes from '../router/Routes'
-import AppFooter from './AppFooter'
+// import AppFooter from './AppFooter'
```
remove comment please

VolodymyrKholiavko commented on src/App/AppContainer.tsx:
```diff
@@ -16,7 +16,7 @@ const AppContainer = () => {
             <Routes />
           </Box>
-          <AppFooter />
+          {/* <AppFooter /> */}
```
remove comment please

VolodymyrKholiavko commented on src/App/AppHeader/Burger.tsx:
```diff
@@ -0,0 +1,33 @@
+  return (
+    <>
+      {isOpen ? (
+        <>
```
render could work without React.Fragment </>

VolodymyrKholiavko commented on src/App/AppHeader/LanguageSelect.tsx:
```diff
@@ -29,21 +33,26 @@ const LanguageSelect = () => {
+            fontWeight={400}
+            fontFamily={'Rubik'}
+          >
+            {text ? text : selectedLanguage}
```
text && selectedLanguage

VolodymyrKholiavko commented on src/modules/Basket/Basket.tsx:
```diff
@@ -46,38 +50,45 @@ const Basket = () => {
+          }}
+        >
+          <ModalBody pl={'0px'} pt={'0px'} pr={'0px'} pb={'0px'}>
+            {selectedBasketType === 'basket' && (
```
may be simplified by switch case and pass as variable in to curly braces

VolodymyrKholiavko commented on src/modules/Basket/DeliveryForm.tsx:
```diff
@@ -76,91 +76,188 @@ const DeliveryForm = ({ setSelectedBasketType }: Props) => {
+          <Stack direction="column" spacing={'3px'}>
+            <Radio
+              style={{
+                borderColor: deliveryType === 'pickup' ? 'black' : 'grey',
```
let out as const and pass here

VolodymyrKholiavko commented on src/modules/Basket/DeliveryForm.tsx:
```diff
@@ -76,91 +76,188 @@ const DeliveryForm = ({ setSelectedBasketType }: Props) => {
+            </Radio>
+            <Radio
+              style={{
+                borderColor: deliveryType === 'delivery' ? 'black' : 'grey',
```
let out as const and pass here

VolodymyrKholiavko commented on src/modules/Basket/InfoToPay.tsx:
```diff
@@ -1,34 +1,102 @@
-          fontWeight={600}
-          decoration={isVoucherActive ? 'line-through' : 'none'}
+          fontFamily={'Rubik'}
+          color={isVoucherActive || isDiscounted ? '#9090A4' : '#002034'}
```
declare a condition and pass as variable here

VolodymyrKholiavko commented on src/modules/Basket/InfoToPay.tsx:
```diff
@@ -1,34 +1,102 @@
+          fontFamily={'Rubik'}
+          color={isVoucherActive || isDiscounted ? '#9090A4' : '#002034'}
+          fontWeight={400}
+          decoration={isVoucherActive || isDiscounted ? 'line-through' : 'none'}
```
declare a condition and pass as variable here

VolodymyrKholiavko commented on src/modules/Home/components/HomeContent.tsx:
```diff
@@ -56,7 +66,7 @@ const HomeContent = () => {
         )}
       </Container>
-      <GratitudeNote />
+      {/* <GratitudeNote /> */}
```
remove comment please

VolodymyrKholiavko reviewed (changes requested):
done

YelyzavetaDemchenko committed:
feature/basketRedesign

Added scrolling when the modal is opened, refactored the code according to revue.
